agent/src/Factories/BrowserFactory.php

```diff
@@ -29,7 +29,8 @@ public function __invoke(
 
         $browser = (new ReactBrowser($connector))
             ->withTimeout($timeout)
-            ->withBase($baseUrl);
+            ->withBase($baseUrl)
+            ->withoutHeader('User-Agent');
```

Comment:
This isn't explicitly required, however it reduces the amount of work performed on each request when it tries to merge the default headers and the request instance headers.